### ISSUE TITLE
A bug with function "CompCCAT"

### ISSUE THREAD

**Issue by zyb1984** [aet21/SCENT#2]
WIth orginal code 
> library(SCENT);
>  data(net13Jun12);
>  data(dataChu)
>  lscChu0.m <- log2(scChu.m+1);
>  ccat.v <- CompCCAT(exp = lscChu0.m, ppiA = net13Jun12.m); 
   With error message "Error in if (classMATRIX == "matrix") { : the condition has length > 1" 
 >  class(lscChu0.m)
##  [1] "matrix" "array" 
>  class(net13Jun12.m)
## [1] "matrix" "array"  
        And what happened? why with this error message

**Comment by aet21** [comment]
Yes, this bug is caused by the more recent version of R which assigns a vector to class(matrix), with the 2nd lelement of this vector being "array". Not sure why R has done this. I have debugged this now.

**Comment by zyb1984** [comment]
Thank you very much for you fixing this error.
